app/src/main/java/com/example/timeman/MainActivity.kt

```diff
@@ -5,14 +5,7 @@ import android.content.Context
 import android.content.Intent
 import android.content.IntentFilter
 import android.os.Bundle
-import com.google.android.material.snackbar.Snackbar
 import androidx.appcompat.app.AppCompatActivity
-import androidx.navigation.findNavController
-import androidx.navigation.ui.AppBarConfiguration
-import androidx.navigation.ui.navigateUp
-import androidx.navigation.ui.setupActionBarWithNavController
-import android.view.Menu
-import android.view.MenuItem
```

Comment:
removed unneeded clutter - https://kotlinlang.org/docs/basic-syntax.html#ranges

Reply:
Android Studio can do this for you automatically, with Control Shift O.

Reply:
The suggestion has been resolved thank you